RELEASE.md

```diff
@@ -0,0 +1,3 @@
+release type: patch
+
+This release integrates `strawberry-graphql-django` package into strawberry core so that it is possible to use django package directly through `strawberry.django` namespace.
```

Comment:
```suggestion
Integrate the `strawberry-graphql-django` package into Strawberry core so that it's 
possible to use the Django extension package directly through the `strawberry.django`
namespace.
```